themes/default/content/what-is/what-is-devops.md

```diff
@@ -0,0 +1,61 @@
+
+- **Continuous integration and continuous delivery (CI/CD)**: Continuous integration (CI) and continuous delivery (CD) are interconnected practices that streamline the software development process. CI involves developers frequently merging code changes into a central repository where automated builds and tests are conducted to identify and fix bugs swiftly, thus enhancing software quality and shortening the update validation period. CD builds on CI's foundation, automatically building, testing, and preparing code for release. It further automates the deployment of these changes to various testing or production environments, ensuring that the code is not just ready for deployment but is continuously delivered to production with minimal manual intervention. This creates a deployment-ready build artifact that has been rigorously tested, facilitating frequent and stable releases, with monitoring systems in place to ensure transparency throughout the process. [Learn more about Pulumi’s CI/CD integration](https://www.pulumi.com/docs/guides/continuous-delivery/).
+
+- **Version control and change management**: Version control is crucial in DevOps, not just for managing software code but also for deploying using infrastructure as code (IaC). By applying version control to IaC processes, teams gain the ability to track changes, collaborate effectively, and quickly revert to previous states of the infrastructure, ensuring consistency and reliability in both application development and the environments where these applications are deployed. This unified approach to version control across software and infrastructure fosters better alignment, efficiency, and quality assurance in the delivery pipeline. Learn more about [Version control in Pulumi](https://www.pulumi.com/docs/intro/concepts/state/#backends).
+
+- **Testing infrastructure as code**: [Testing infrastructure as code](https://www.pulumi.com/docs/using-pulumi/testing/) (IaC) in DevOps is similar to testing applications using standard testing tools for various test types. Unit tests swiftly validate code segments without real infrastructure, relying on the same programming language as the IaC programs. Property tests take a step further by provisioning real infrastructure to check resource configurations before and after deployment. Integration tests comprehensively establish a temporary infrastructure setup to verify end-to-end functionality, particularly useful in CI/CD workflows. For more robust testing, one might employ fuzz testing to challenge the infrastructure with unexpected inputs, chaos testing to assess resilience by intentionally causing disruptions, and stress testing to push the system to its limits. These advanced tests, enabled by the flexibility of using general-purpose languages for IaC, provide a wide array of tools to ensure the infrastructure's integrity and robustness.
```

Comment:
this is actually different from the other bullets where you put the Pulumi related article as part of the Learn more